packages/config/config/devices/0x031e/lzw31-sn.json

```diff
@@ -89,83 +88,81 @@
 		},
 		{
 			"#": "5",
-			"label": "Minimum Dim Level",
-			"unit": "%",
-			"valueSize": 1,
-			"minValue": 1,
-			"maxValue": 45,
-			"defaultValue": 1
+			"$import": "~/templates/master_template.json#minimum_dim_level_1-99",
+			"maxValue": 45
 		},
 		{
 			"#": "6",
-			"label": "Maximum Dim Level",
-			"unit": "%",
-			"valueSize": 1,
-			"minValue": 55,
-			"maxValue": 99,
-			"defaultValue": 99
+			"$import": "~/templates/master_template.json#maximum_dim_level_1-99",
+			"minValue": 55
 		},
 		{
 			"#": "7",
-			"label": "Invert Switch",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 1,
-			"defaultValue": 0,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "Disabled",
-					"value": 0
-				},
-				{
-					"label": "Enabled",
-					"value": 1
-				}
-			]
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Invert Switch"
 		},
 		{
 			"#": "8",
-			"label": "Auto Off Timer",
+			"label": "Auto-off Timer",
 			"unit": "seconds",
 			"valueSize": 2,
 			"minValue": 0,
 			"maxValue": 32767,
-			"defaultValue": 0
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
 		},
 		{
 			"#": "9",
-			"label": "Default Level",
-			"unit": "%",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 99,
+			"label": "Default Level (Manual)",
+			"$import": "~/templates/master_template.json#base_0-99_nounit",
```

Comment:
Please make sure the `$import` comes before the label, otherwise we risk overwriting the label if the template is ever changed.